data/saas/config/mailchimp_config.yml

```diff
@@ -69,7 +69,22 @@ saas_config:
               type: query
               identity: email
               data_type: string
-          data_path: exact_matches/members
+          data_path: exact_matches.members
+        update:
+          path: /3.0/lists/<list_id>/members/<subscriber_hash>
+          request_params:
+            - name: list_id
+              type: path
+              references:
+                - dataset: mailchimp_connector_example
+                  field: member.list_id
+                  direction: from
```

Comment:
All your references are in the "from" direction, but they can also be specified to point the other way, does this affect your implementation?

Reply:
The way I understood direction to work is that if we specify the direction as `from` in the SaaS config, then the SaaS connector nodes would receive the necessary data from the referenced datasets as part of the graph task execution. I didn't see a need to specify any references as `to` since those datasets could just reference the SaaS dataset using a `from` reference. Is this a valid understanding/assumption?

Reply:
You can use `from`, my question to rephrase is, does it break if someone tries to use `to` with Saas configs, our other datasets have both `to` and `from` directions between their collections, but is it a requirement for saas configs that only the "from" direction is used?

Reply:
I'll add this to #182 since that's where I have the endpoints/tests.

Reply:
Merged in validation from main branch.

Reply:
thanks @galvana